src/httprpc.cpp

```diff
@@ -322,3 +322,8 @@ void StopHTTPRPC()
         httpRPCTimerInterface.reset();
     }
 }
+
+const std::set<std::string>& GetWhitelistedRpcs(const std::string& userName)
+{
+    return g_rpc_whitelist[userName];
```

Comment:
```suggestion
    return g_rpc_whitelist.at(userName);
```

Otherwise there is no guarantee that the whitelist will not get corrupted

Reply:
Thanks!